cfme/base/login.py

```diff
@@ -103,7 +103,6 @@ def logged_out(self):
 
     def logout(self):
         self.settings.select_item('Logout')
-        self.browser.handle_alert(wait=None)
```

Comment:
https://github.com/ManageIQ/integration_tests/pull/8848#discussion_r288146353

Reply:
Why do we have that `handle_alert`? I don't remember seeing any alert box on logging out.